### PR TITLE
AN-2245 superfluid staked balances

### PR DESCRIPTION
**Pull request by eric-laurello**

### CODE REVIEW

File: models/silver/silver__locked_liquidity_actions.sql [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+        A.msg_group,
+        {# msg_index, #}
+        A.msg_type,
+        A.lock_id,
+        {# j, #}
+        b.attribute_value AS action,
+        CASE
+            WHEN b.attribute_value = '/osmosis.lockup.MsgExtendLockup' THEN 'extend lockup'
+            WHEN A.msg_type = 'unpool_pool_id' THEN 'unpool'
+            WHEN j :"lock_tokens--duration" IS NOT NULL THEN 'inital lock'
```
Comment (desmond-hui) [inline models/silver/silver__locked_liquidity_actions.sql:162]: typo for the value

File: models/silver/silver__locked_liquidity_balances.sql [hunk 2]
```diff
@@ -0,0 +1,180 @@
+{{ config(
+    materialized = 'incremental',
+    unique_key = "_unique_key",
+    incremental_strategy = 'merge',
+    cluster_by = ['block_timestamp::DATE'],
+) }}
```
Comment (desmond-hui) [inline models/silver/silver__locked_liquidity_balances.sql:6]: Add business logic output testing 

see: https://github.com/FlipsideCrypto/solana-models/blob/56d024d15592dabe5a657c699322d2ceaafac699/models/silver/nfts/silver__nft_mints.yml#L11

**Review by desmond-hui — CHANGES_REQUESTED** [review]
Add `.DS_Store` to `.gitignore`

**Comment by eric-laurello** [comment]
@desmond-hui requested changes applied

**Review by desmond-hui — APPROVED** [review]

**Pull request merged**